src/main/java/com/synopsys/integration/alert/channel/azure/boards/AzureBoardsChannel.java

```diff
@@ -74,8 +79,22 @@ public IssueTrackerResponse sendRequests(IssueTrackerContext context, List<Issue
     }
 
     private IssueConfig createIssueConfig(FieldAccessor fieldAccessor) {
-        // FIXME implement
-        return null;
+        String azureProjectName = fieldAccessor.getStringOrNull(AzureBoardsDescriptor.KEY_AZURE_PROJECT);
+        String workItemCreatorEmail = fieldAccessor.getStringOrNull(AzureBoardsDescriptor.KEY_WORK_ITEM_CREATOR_EMAIL);
+        String workItemTypeName = fieldAccessor.getStringOrNull(AzureBoardsDescriptor.KEY_WORK_ITEM_TYPE);
+        boolean commentOnWorkItems = fieldAccessor.getBooleanOrFalse(AzureBoardsDescriptor.KEY_WORK_ITEM_COMMENT);
+        String completedStateName = fieldAccessor.getStringOrNull(AzureBoardsDescriptor.KEY_WORK_ITEM_COMPLETED_STATE);
+        String reopenStateName = fieldAccessor.getStringOrNull(AzureBoardsDescriptor.KEY_WORK_ITEM_REOPEN_STATE);
+        return new IssueConfig(
+            azureProjectName,
+            azureProjectName,
```

Comment:
should azureProjectName be here twice? Are projectName and projectKey the same?

Reply:
The intention of this class was not clear. The three fields are: 
- projectKey 
- projectName 
- projectId 

There is no concept of projectKey vs projectName in Azure. There is a projectId (UUID type), but it would have to be requested through the API.